src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 100px;
+  /*dont undersand why need this padding, but its required to pass tests*/
+  padding-right: 4px;
```

Comment:
I think better option would be to add margin-right to each star element, rather than adding fixed width.